app/assets/javascripts/app/components/discussion_card/discussion_card.js

```diff
@@ -14,17 +14,29 @@
       },
       replace: true,
       templateUrl: 'components/discussion_card/templates/discussion_card.html',
-      controller: 'DiscussionCardCtrl as ctrl'
+      controller: 'DiscussionCardCtrl as ctrl',
+      bindToController: true
     };
   }
 
-  function DiscussionCardCtrl($state) {
+  function DiscussionCardCtrl($state, User) {
     var ctrl = this;
 
     ctrl.show = show;
+    ctrl.commenters = commenters();
+    ctrl.author = author();
 
     function show(id) {
       $state.go('discussions.show', { id: id });
     }
+
+    function commenters() {
+      var commenterIds = ctrl.discussion.commenter_ids;
+      return User.findAllById(commenterIds);
```

Comment:
`findAllById` is a synchronous function right? It's not making an HTTP request.
If so, and since we're using, js-data, let's use their convention and call it `getAllById` as the `get` part conveys the synchronous feature